components/v1/Navigation.tsx

```diff
@@ -21,13 +22,20 @@ type NavigationProps = {
  */
 const Navigation: FunctionComponent<NavigationProps> = ({ marketsActive, portfolioActive }) => {
     const [scrollPosition, setScrollPosition] = useState(0);
+    const router = useRouter();
+
+    const handleScroll = () => {
+        const position = window.pageYOffset;
+        setScrollPosition(position);
+    };
```

Comment:
if this handleScroll not used outside of useEffect, then just place it inside of useEffect

Reply:
fixed sir